Makefile

```diff
@@ -42,6 +42,8 @@ $(PACKAGE_SCRIPTS) \
 --config-files "/etc/$(BINARY)/$(CONFIG_FILE)"
 endef
 
+VERSION_LDFLAGS:=-X $(POLLER_PATH).Branch=$(BRANCH) -X $(POLLER_PATH).BuildDate=$(DATE) -X $(POLLER_PATH).Revision=$(COMMIT) -X $(POLLER_PATH).Version=$(VERSION)-$(ITERATION)
```

Comment:
Couldn't we just do `-X github.com/prometheus/common/version.Branch=$(BRANCH)` and repeat for the others? Then we don't need new/more global variables, and we could in fact remove the `Version` variable by doing it this way.  Then there's only two new lines of Go code, the rest is in the Makefile. Thoughts?

Reply:
Also, please add a `\` or two to wrap this long line.

Reply:
We could go directly to the version package variable. But sometimes it's nicer to have it in the local vars for other uses like printing it out in the logs. I just didn't add that feature here.

Reply:
So, when I try and use `-X github.com/prometheus/common/version...`, it doesn't seem to populate correctly. I think I'd rather stick to the current method of getting these variables set.

But I split up the long line.

Reply:
This worked, but I don't know how portable it is if we move off of `dep` and into the module world.
```
-X github.com/davidnewhall/unifi-poller/vendor/github.com/prometheus/common/version.Version=$(VERSION)-$(ITERATION)
```

Reply:
Ahh, right! That jogged my memory from our setup before our go mod changes in Prometheus land.

With go mod, it's simplifies to `-X github.com/prometheus/common/version.Version`